src/selectedtests/app/app.py

```diff
@@ -25,14 +34,30 @@ def create_app() -> Flask:
         doc="/swagger",
     )
 
+    # Creating index no-ops if index already exists
+    setup_indexes(mongo.test_mappings_queue())
+
     add_health_endpoints(api)
+    add_project_test_mappings_endpoints(api, mongo, evg_api)
 
     return app
 
 
+def _get_mongo_wrapper() -> MongoWrapper:
+    """
+    Get an instance of the mongo wrapper based on environment variables.
+
+    :return: MongoWrapper instance.
+    """
+    mongo_uri = os.environ.get("SELECTED_TESTS_MONGO_URI")
```

Comment:
I have added the `SELECTED_TESTS_MONGO_URI` secret to the `selected-tests` namespace in helm, accessible through:
```
helm ksec get selected-tests
```